### ISSUE TITLE
Some recent change broke Octave simulations

### ISSUE THREAD

**Issue by kupiqu** [DynaSim/DynaSim#347]
Some recent change in dsSimulate breaks simulations in Octave.

This is what happens when running dsDemos:

```
dsDemos
sh: 1: Syntax error: "(" unexpected
Error: structure has no member 'pop1_s'
         in solve_ode_20171218234822_849 (line 32)
         in dsSimulate>tryFn (line 1023)
         in dsSimulate (line 772)
         in dsDemos (line 87)
structure has no member 'pop1_s'
error: called from
    solve_ode_20171218234822_849 at line 32 column 12
    dsSimulate>tryFn at line 1023 column 48
    dsSimulate at line 772 column 5
    dsDemos at line 87 column 5
```

**Comment by kupiqu** [comment]
The issue doesn't seem to come from dsSimulate though. @jsherfey any clue where may this issue with `pop1_s` come from ?

**Comment by kupiqu** [comment]
Ok, found the issue (actually introduced by myself, my bad!), working on a fix

**Comment by kupiqu** [comment]
Fixed with merge pull request #350
